packages/bpk-mixins/src/mixins/_buttons.scss

```diff
@@ -185,6 +185,45 @@
   }
 }
 
+/// Outline button. Modifies the bpk-button & bpk-button--secondary mixins.
+///
+/// @require {mixin} bpk-button
+/// @require {mixin} bpk-button--secondary
+///
+/// @example scss
+///   .selector {
+///     @include bpk-button();
+///     @include bpk-button--secondary();
```

Comment:
in the `BpkButton.scss` we aren't using the `@include bpk-button--secondary()` mixin so should this comments be updated to reflect this?

Reply:
Another good catch :)